<launch>
    <param name="MODE" type="int" value="1" />
    <param name="ROBOTS_NUM" type="double" value="6" />
    <param name="ROBOT_FOV" type="double" value="150.0" />
    <param name="ROBOT_RANGE" type="double" value="15.0" />
    <param name="AREA_SIZE_x" type="double" value="40.0" />
    <param name="AREA_SIZE_y" type="double" value="40.0" />
    <param name="AREA_LEFT" type="double" value="-20.0" />
    <param name="AREA_BOTTOM" type="double" value="-20.0" />
    <param name="GRAPHICS_ON" type="bool" value="true" />


    <node name="pf_coverage_0"  pkg="pf_coverage" type="pf_coverage" output="screen">
        <rosparam>
            MODE: 1
            ROBOTS_NUM: 6
            ROBOT_ID: 0
            ROBOT_RANGE: 15.0
            ROBOT_FOV: 150.0
            AREA_SIZE_x: 40.0
            AREA_SIZE_y: 40.0
            AREA_LEFT: -20.0
            AREA_BOTTOM: -20.0
            GRAPHICS_ON: true
            GOAL_X: 4.0
            GOAL_Y: -4.0
        </rosparam>
    </node>

    <node name="pf_coverage_1"  pkg="pf_coverage" type="pf_coverage" output="screen">
        <rosparam>
            MODE: 1
            ROBOTS_NUM: 6
            ROBOT_ID: 1
            ROBOT_RANGE: 15.0
            ROBOT_FOV: 150.0
            AREA_SIZE_x: 40.0
            AREA_SIZE_y: 40.0
            AREA_LEFT: -20.0
            AREA_BOTTOM: -20.0
            GRAPHICS_ON: true
            GOAL_X: 4.0
            GOAL_Y: 4.0
        </rosparam>
    </node>

    <node name="pf_coverage_2"  pkg="pf_coverage" type="pf_coverage" output="screen">
        <rosparam>
            MODE: 1
            ROBOTS_NUM: 6
            ROBOT_ID: 2
            ROBOT_RANGE: 15.0
            ROBOT_FOV: 150.0
            AREA_SIZE_x: 40.0
            AREA_SIZE_y: 40.0
            AREA_LEFT: -20.0
            AREA_BOTTOM: -20.0
            GRAPHICS_ON: true
            GOAL_X: -4.0
            GOAL_Y: 4.0
        </rosparam>
    </node>

    <node name="pf_coverage_3"  pkg="pf_coverage" type="pf_coverage" output="screen">
        <rosparam>
            MODE: 1
            ROBOTS_NUM: 6
            ROBOT_ID: 3
            ROBOT_RANGE: 15.0
            ROBOT_FOV: 150.0
            AREA_SIZE_x: 40.0
            AREA_SIZE_y: 40.0
            AREA_LEFT: -20.0
            AREA_BOTTOM: -20.0
            GRAPHICS_ON: true
            GOAL_X: -4.0
            GOAL_Y: -4.0
        </rosparam>
    </node>

    <node name="pf_coverage_4"  pkg="pf_coverage" type="pf_coverage" output="screen">
        <rosparam>
            MODE: 1
            ROBOTS_NUM: 6
            ROBOT_ID: 4
            ROBOT_RANGE: 15.0
            ROBOT_FOV: 150.0
            AREA_SIZE_x: 40.0
            AREA_SIZE_y: 40.0
            AREA_LEFT: -20.0
            AREA_BOTTOM: -20.0
            GRAPHICS_ON: true
            GOAL_X: 0.0
            GOAL_Y: -6.0
        </rosparam>
    </node>

    <node name="pf_coverage_5"  pkg="pf_coverage" type="pf_coverage" output="screen">
        <rosparam>
            MODE: 1
            ROBOTS_NUM: 6
            ROBOT_ID: 5
            ROBOT_RANGE: 15.0
            ROBOT_FOV: 150.0
            AREA_SIZE_x: 40.0
            AREA_SIZE_y: 40.0
            AREA_LEFT: -20.0
            AREA_BOTTOM: -20.0
            GRAPHICS_ON: true
            GOAL_X: 0.0
            GOAL_Y: 6.0
        </rosparam>
    </node>
</launch>
